src/test/java/uk/gov/digital/ho/proving/income/audit/statistics/PassRateStatisticsServiceTest.java

```diff
@@ -50,6 +50,7 @@ public void setUp() {
 
     /**************************
```

Comment:
If you need to document groups of tests in the file using comments, it might be better to split the test into different files.